### PR TITLE
Website dev

### PR DESCRIPTION
**Pull request by kaitj**
Looking for some comments and insights into modifications for the site.

Will note that the site is probably not mobile friendly, made no implementations for different screen sizes.

### CODE REVIEW

**Comment by tkkuehn** [comment]
Looking good so far! I've got some time today, so I'm going to take a crack at cleaning up the templates a bit by using some [template inheritance](https://jinja.palletsprojects.com/en/2.10.x/templates/#template-inheritance), and replacing some `div`s with more [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#Semantics_in_HTML) elements.

**Comment by Park-Patrick** [comment]
looks good to me!

**Comment by kaitj** [comment]
The .gif replacement from @Park-Patrick needs to be fixed. Currently on top of the navigation bar on my screen.

**Comment by tkkuehn** [comment]
> 
> 
> The .gif replacement from @Park-Patrick needs to be fixed. Currently on top of the navigation bar on my screen.

Fixed this, I think. A bit of the custom CSS caused the problem.

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    </nav>
+    <!-- Should find a way to add this line in CSS -->
+    <hr>
+  </header>
+
+  <main>{% block content %}{% endblock %}</main>
+
+  <footer>
+    <div class="copyright">
+      &copy; <script type="text/javascript">document.write(new Date().getFullYear());</script> Anatomical Fiducials
```
Comment (tkkuehn) [inline templates/base.html]: The footer generally looks good, but is there a point to this copyright statement? I dunno if any of this stuff is copyrighted, and I don't think Anatomical Fiducials is an entity that can hold a copyright...

Comment (kaitj) [reply templates/base.html]: Fair point re afids as an entity with ability to hold copyright.

With regards to legal/copyright/license stuff, my knowledge is limited at best and hopefully someone else knows more / there is someone to talk to about that stuff. If anything, rights would go to Jon? There are definitely a few things on the website that we need to figure out. For example, we have the HBM cover in our banner - I believe we can use that, as well as the actual protocol.

Anyways, I think this is a point that needs more looking into / discussion.

Comment (jclauneuro) [reply templates/base.html]: There should not be any copyright issues with the AFIDs protocol itself. The whole point was to build it in an open way and it was all released as a preprint prior to submission to HBM.

Note that HBM has recently switched to an open access model. I just checked and as promised the AFIDs article is now available open access: https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.24693. The only thing I'm not sure about is with regards to licensing for the cover image.

**Comment by tkkuehn** [comment]
In the interest of keeping things simple, I've gotten rid of the milligram CSS framework and migrated anything that depended on it to the Bootstrap equivalent - As a nice bonus, the validator behaves a bit better on mobile now, too.

**Comment by kaitj** [comment]
Looks good! Only thing is the footer - it looks like it resizes on its own and the github and twitter icons end up on the same line as the text at full screen. Other then that I think it's pretty much ready to merge once we have the landing page stuff (#64)

**Comment by tkkuehn** [comment]
> Only thing is the footer - it looks like it resizes on its own and the github and twitter icons end up on the same line as the text at full screen.

What's the expected behaviour? Shouldn't be hard to adjust if we prefer a different arrangement.

**Comment by kaitj** [comment]
> What's the expected behaviour? Shouldn't be hard to adjust if we prefer a different arrangement.

I think the one that I've seen most is to have the text on one line and then the icons on another. As for the order, not entirely sure.

**Comment by tkkuehn** [comment]
I added a brief placeholder intro to replace the lorem ipsem that was there before, so I think we can get this merged now.

**Comment by tkkuehn** [comment]
We're going to take away the "View DB" and "Analytics" buttons for now - they don't really add any value at this point.

**Comment by kaitj** [comment]
Merging current PR as is in order to push updates along. See #66 for removal of "View DB" and "Analytics" buttons.

**Pull request merged**